src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Main {
+    static final Random random = new Random();
+
+    public static void main(String[] args) {
+        ColorSupplier colorSupplier = new ColorSupplier();
+
+        Figure circle = new Circle();
+        Figure rectangle = new Rectangle();
+        Figure rightTriangle = new RightTriangle();
+        Figure square = new Square();
+        Figure isoscelesTrapezoid = new Figure();
+        Figure[] figures = new Figure[]{circle, rectangle, rightTriangle, square, isoscelesTrapezoid};
+        int count = 1;
```

Comment:
The variable `count` is declared but not used anywhere in the code. If it's not necessary, consider removing it to avoid unused variables in the codebase.